tests/unit/linters/test_base.py

```diff
@@ -57,3 +58,27 @@ def test_linter_issue_string_no_url(self, linter_issue):
     def test_linter_issue_string_filename_no_url(self, linter_issue):
         issue = linter_issue(filename="foo.txt", url=None)
         assert f"{issue}" == "test: foo.txt: Linter message text"
+
+
+def test_lint_ignore_api():
+    """Test projects.Lint "ignore"-related API."""
```

Comment:
the API is rather easy to follow, but this test is hard to unpack. Maybe splitting it up a bit can add clarity

Reply:
Hmm yes I see what you mean. I split this and reworded the contents a bit, please take a look